Target Search.md

```diff
@@ -0,0 +1,64 @@
+# Summary
+Group members collectively guess coordinates, and the value of the experimenters diagram determines the value of their decision. There were undifferentiated and differentiated power structure conditions.
+
+# References
+Main Paper: http://collections.uakron.edu/digital/collection/p15960coll1/id/21257/
+
+Task Paper: https://psycnet-apa-org.proxy.library.upenn.edu/fulltext/1960-05749-001.pdf
+
+# Stimuli
+## The visual components
+![89](images/89.png)
+
+## Materials for alternative versions of the experiment 
+![90](images/90.png)
+![91](images/91.png)
+![92](images/92.png)
+![93](images/93.png)
+
+# Procedure
+## Steps
+>First, the participants completed a 24-item IP scale which contained items like "In meetings of groups to which I belong I often find myself trying to get things organized" to determine IP tendencies
+>
+>Then, one member of each group was required to volunteer stand before the group and record on the blackboard the group's decisions. This validated the IP scale.
+>
+>Paricipants were then assigned to the various groups and conditions in haphazard order.
+>
+>The group of four is seated infront of a blackboard with a blank diagram with letters on the x-axis and numbers on the y-axis.
+>
+>Participants are read the instructions.
+>
+>Paricipants will be given 3 minutes for discussion before making a group decision, then will have 10 trials to choose a coordinate on each of the five problems.
+>
+>After each decision, they will be told their score based on the diagram.
+>
+>The highest possible score would be a score of 200, where the group chooses the central coordinate on all 10 trials.
+>
+>At the end of each problem, the participants could examine the experimenters diagram. They were then asked to complete a questionaire that required ratings of 1. personal satisfaction with the job, 2. group cooperation, and 3. group performance. They were also asked to rank members of the group in order of amount of influence on the group's decisions.
+
+## Roles 
+The group has a blank diagram, while the experimenter has a diagram with target values written in. The experimenter marked the groups decision on the blackboard and entered the score earned.
+
+An observer was also persent who sat at one side of the room and unobtrusively recorded selected aspects of the group interaction.
+
+The experimenter and observer separately ranked the participants in order of perceived influence on the group process.
+
```

Comment:
In the second test, 20 four-person groups were observed in a situation that required that one member of each group volunteer for a task that would make him stand out from the other members. In addition to functioning as a group member, the volunteer was required to stand before the group and record on the blackboard the group's decisions regarding a number of issues.